data/available_datasets.json

```diff
@@ -1,6 +1,162 @@
 {
-    "last_updated": "2023-01-13",
+    "last_updated": "2023-02-01",
     "datasets": [
+        {
+            "doi": "10.1016/j.concog.2023.103464",
+            "available_items": [
+                {
+                    "type": "data",
+                    "repo_name": "OSF",
+                    "accession_number": "",
+                    "description": "Students survey data",
```

Comment:
can you mention something about the variables in the data like 'Survey data from students tested on cognition' (I don't know what the data is, just something like that